specification.md

```diff
@@ -897,13 +897,40 @@ debugging purposes, but which should not, in general, be presented to an
 end-user.
 
 
-<a name="invalidjobexception-getjob"></a>
+
+### SubmitException
+
+This exception is thrown when the
+[`JobExecutor.submit()`](#jobexecutor-submit) call fails for a reason
+that is independent of the job being submitted.
+
+#### Methods
+
+<a name="submitexception-getdetail"></a>
 ```java
-Job getJob()
+FaultDetail getDetail()
```

Comment:
Do we have an issue open about detailing `FaultDetail`?  I see it is also used for `InvalidJobException`.  So adding it here is fine with me, but I just want to make sure we have that as a recorded TODO.

Reply:
PS - a search for `faultdetail` returns nothing in our issue tracker